js/thumbnail-rendering.js

```diff
@@ -8,12 +10,14 @@ const pictureTemplate = document.querySelector('#picture')
 const renderMiniatures = (thumbnailPhotos) => {
   const picturesThumbnailFragment = document.createDocumentFragment();
 
-  thumbnailPhotos.forEach(({url, likes, comments}) => {
+  thumbnailPhotos.forEach((thumbnailPhoto) => {
+    const {url, likes, comments} = thumbnailPhoto;
     const pictureElement = pictureTemplate.cloneNode(true);
     pictureElement.querySelector('.picture__img').src = url;
     pictureElement.querySelector('.picture__likes').textContent = likes;
     pictureElement.querySelector('.picture__comments').textContent = comments.length;
     picturesThumbnailFragment.appendChild(pictureElement);
+    showBigPicture(thumbnailPhoto);
```

Comment:
Тут не надо просто открывать изображение. Нужно вешать обработчик. Что-то типа
```suggestion
    pictureElement.addEventListener('click', () =>showBigPicture(thumbnailPhoto));
```